packages/vue/src/components/organisms/SfSidebar/SfSidebar.vue

```diff
@@ -22,25 +21,47 @@
               class="sf-heading--left sf-heading--no-underline sf-sidebar__title desktop-only"
             />
           </slot>
+          <slot
+            v-if="position === 'right'"
+            name="icon"
+            v-bind="{ close, button }"
+          >
+            <SfIcon
+              v-if="button"
+              icon="cross"
+              size="14px"
+              role="button"
+              aria-label="close"
+              class="sf-sidebar__icon desktop-only"
+              @click="close"
+            />
+          </slot>
+          <slot v-else name="circle-icon" v-bind="{ close, button }">
+            <SfCircleIcon
+              v-if="button"
+              icon-size="14px"
+              icon="cross"
+              class="sf-sidebar__circle-icon desktop-only"
+              @click="close"
+            />
+          </slot>
+        </div>
+        <!--@slot Use this slot to place content inside the modal bar.-->
+        <div ref="content" class="sf-sidebar__content">
           <slot />
         </div>
-        <slot name="circle-icon" v-bind="{ close, button }">
-          <SfCircleIcon
-            v-if="button"
-            icon-size="14px"
-            icon="cross"
-            icon-color="white"
-            class="sf-sidebar__circle-icon desktop-only"
-            @click="close"
-          />
-        </slot>
+        <SfSticky class="sf-sidebar__bottom">
```

Comment:
why do you use SfSticky here? it just set `position: sticky` and when it's empty it shouldn't bee display, it should be optional :)